docs/web/docs/3-reference/6-aws_hub_and_spoke_roles.mdx

```diff
@@ -93,41 +116,52 @@ directly change the IAMbic configuration to specify the arn of the spoke role.
 
 ```json
 {
-    "Version": "2012-10-17",
-    "Statement": [
-        {
-            "Action": [
-                "identitystore:Describe*",
-                "identitystore:Get*",
-                "identitystore:List*",
-                "organizations:describe*",
-                "organizations:list*",
-                "iam:Get*",
-                "iam:List*",
-                "sso:Describe*",
-                "sso:Get*",
-                "sso:List*",
-                "sso:Search*"
-            ],
-            "Resource": [
-                "*"
-            ],
-            "Effect": "Allow"
-        },
-        {
-            "Action": [
-                "secretsmanager:CreateSecret",
-                "secretsmanager:GetSecretValue",
-                "secretsmanager:describesecret",
-                "secretsmanager:listsecrets",
-                "secretsmanager:listsecretversionids",
-                "secretsmanager:PutSecretValue"
-            ],
-            "Resource": [
-                "arn:aws:secretsmanager:*:*:secret:iambic-config-secrets-*"
-            ],
-            "Effect": "Allow"
-        }
-    ]
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Action": [
+        "identitystore:Describe*",
+        "identitystore:Get*",
+        "identitystore:List*",
+        "organizations:describe*",
+        "organizations:list*",
+        "iam:Get*",
+        "iam:List*",
+        "sso:Describe*",
```

Comment:
We have sso:* in the hub and spoke roles doc above

Reply:
The IambicSpokeRoleReadOnly is intentionally slimmer from https://github.com/noqdev/iambic/pull/317

The regular `IambicSpokeRole` will have read + write on the sso  whereas the `IambicSpokeRoleReadOnly` will have read-only access to `iam` and `sso` service.

Reply:
ahhh, missed that, thanks for clarifying!